docs/prometheus.rst

```diff
@@ -0,0 +1,39 @@
+.. highlight:: python
+
+
+==========
+Prometheus
+==========
+
+Talisker provides optional `prometheus_client` integration and configuration.
+
+Installation
+------------
+
+The package supports extras args to install `prometheus_client`::
+
+    $ pip install talisker[prometheus]
+
+Configuration
+-------------
+
+`prometheus_client` integration needs no extra configuration if running
+in single process mode.
+
+If the app is running in `multiprocess mode <https://github.com/prometheus/client_python#multiprocess-mode-gunicorn>`_
+(ie. with multiple workers), the `prometheus_multiproc_dir` envvar should be set
+to a writable directory that can be used for metrics. If this envvar is not found,
+a temporary directory will be created and used instead.
```

Comment:
Another thought - how does prometheus_client behave in a multithreaded environment? Is there locking involved?